frontend/src/App.js

```diff
@@ -9,17 +9,28 @@ import ProtectedComponent from "./components/ProtectedComponent";
 
 import Login from './pages/Login';
 import Profile from './pages/Profile';
+import Customers from "./pages/customer/Customers";
+import {AuthContext} from "./context/authContext";
 
 function App() {
 
+    const {user, setUser} = useContext(AuthContext);
+    const currentCustomerId = user && user.currentCustomerId ? user.currentCustomerId : 0;
+
     return (
         <UserContext>
             <Header/>
             <Switch>
                 <Route exact path='/' component={Login}/>
-                <Route path={'/profile'} render={() => {
```

Comment:
Please, extract anonymous render function to a constant variable, so each rendering of this component will not create a new function.